scaling_performance/host_practices.adoc

```diff
@@ -57,42 +57,106 @@ include writing a large amount of log data.
 etcd is a distributed key-value store that {product-title} uses for
 configuration.
 
+|======================
+|OpenShift Version |etcd version|storage schema version
+|3.3 and earlier |2.x | v2
+|3.4 and 3.5 |3.x | v2
+|3.6 |3.x | v2 (upgrades)
+|3.6 |3.x | v3 (new installs)
+|======================
+
+etcd 3.x brings with it important scalability and performance improvements that reduce CPU, memory, network and disk requirements for any size cluster.  etcd 3.x also implements a backwards compatible storage API that facilitates a two-step migration of the on-disk etcd database.  For migration purposes, the storage mode used by etcd 3.x in OpenShift 3.5 remained in v2 mode.  As of OpenShift 3.6, new installs will use storage mode v3.  Upgrades from previous versions of OpenShift will NOT automatically migrate data from v2 to v3.  This must be done using supplied playbooks following the documented process.
```

Comment:
s/brings with it/introduces
network, and disk requirements
s/installs/installations
s/OpenShift/{product-title}
{product-title} 3.5
{product-title} 3.6
You must use the supplied playbooks and follow the documented process to migrate the data.

        s/NOT/ _not_